grafana/PostgreSQLDetails.json

```diff
@@ -846,28 +845,74 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(rate(ccp_stat_database_deadlocks{job=\"[[jobname]]\"}[5m]))",
```

Comment:
may it is git or github but at this level this change looks questionable.

Reply:
VSCode auto formats json on save.

Reply:
Looks like this is because the entries are just re-ordered. It's added back later on a different line.